css/quotes.css

```diff
@@ -204,6 +202,11 @@ body {
   border-radius: 50px 0 0 50px;
 }
 
+.button[disabled] {
+  color: gray;
+  background-color: #b4b3b3;
+}
```

Comment:
You changed the color of the button, but the color of the hover is still blue.

Reply:
@hanna446 corrected